neurovault/apps/statmaps/urls.py

```diff
@@ -49,6 +49,9 @@
     url(r'^collections/(?P<cid>\d+|[A-Z]{8})/delete$',
         delete_collection,
         name='delete_collection'),
+    url(r'^collections/(?P<cid>\d+|[A-Z]{8})/download$',
```

Comment:
Well, I think it could be done, but that could be problematic for certain cases. eg. Somebody that wants fastly to download a Collection, if we add the name of the collection, it is quite large in some cases... Not sure about it. Just let me know :)